apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -238,12 +238,10 @@ export class AuthService {
   }
 
   async validateSubscriber(payload: ISubscriberJwt): Promise<SubscriberEntity | null> {
-    const subscriber = await this.getSubscriber({
-      environmentId: payload.environmentId,
+    return await this.subscriberRepository.findOne({
+      _environmentId: payload.environmentId,
       _id: payload._id,
     });
```

Comment:
The `SubscriberJwtPayload` should have the `subscriberId` property so we can refactor to use it, and than keep the caching here.